packages/api-explorer/src/AuthBox.jsx

```diff
@@ -5,14 +5,40 @@ const { Operation } = require('oas');
 
 const SecurityInput = require('./SecurityInput');
 
-function Securities({ authInputRef, operation, onChange, oauth, auth, onSubmit }) {
+function Securities({
+  auth,
+  authInputRef,
+  group,
+  groups,
+  oauth,
+  onChange,
+  onGroupChange,
+  onSubmit,
+  operation,
+}) {
+  function onSelect(event) {
```

Comment:
This will make this code a little cleaner:

```suggestion
  function onSelect({ target }) {
```